### DISCUSSION TITLE
Update 6.14

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#287] [Release]
* fix Android Q crash
* see [v6.13 changelog](https://github.com/TBog/TBLauncher/releases/tag/v6.13)

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/'>Update 6.14</a>.</em>

**Comment by ltguillaume** [comment]
Yep, this fixed the issue with 6.13. Thanks!
